### PR TITLE
zh-CN: Create scalars_and_keras.ipynb

### PR DESCRIPTION
**Pull request by ccw1996**
translate https://github.com/tensorflow/tensorboard/blob/master/docs/scalars_and_keras.ipynb

### CODE REVIEW

File: site/zh-cn/tensorboard/scalars_and_keras.ipynb [hunk 1]
```diff
@@ -0,0 +1,536 @@
...
+        "\n",
+        "# 创建在(-1,1)范围内的随机数作为输入\n",
+        "x = np.linspace(-1, 1, data_size)\n",
+        "np.random.shuffle(x)\n",
+        "\n",
+        "# 生成输出数据\n",
+        "# y = 0.5x + 2 + noise\n",
+        "y = 0.5 * x + 2 + np.random.normal(0, 0.05, (data_size, ))\n",
+        "\n",
+        "# Split into test and train pairs. 将数据切成训练和测试集\n",
```
Comment (Mr-Linus) [inline site/zh-cn/tensorboard/scalars_and_keras.ipynb:177]: 英文请删除。

Comment (ccw1996) [reply site/zh-cn/tensorboard/scalars_and_keras.ipynb]: 已解决，谢谢

File: site/zh-cn/tensorboard/scalars_and_keras.ipynb [hunk 2]
```diff
@@ -0,0 +1,536 @@
...
+        "\n",
+        "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://tensorflow.google.cn/tensorboard/scalars_and_keras\"><img src=\"https://tensorflow.google.cn/images/tf_logo_32px.png\" />在 tensorFlow.google.cn 上查看</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/tensorboard/blob/master/docs/scalars_and_keras.ipynb\"><img src=\"https://tensorflow.google.cn/images/colab_logo_32px.png\" />Google Colab 中运行</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://github.com/tensorflow/tensorboard/blob/master/docs/scalars_and_keras.ipynb\"><img src=\"https://tensorflow.google.cn/images/GitHub-Mark-32px.png\" />在 GitHub 上查看源代码</a>\n",
```
Comment (lamberta) [inline site/zh-cn/tensorboard/scalars_and_keras.ipynb:54]: Thanks. These links should point to this notebook in the docs-l10n repo :)

Comment (ccw1996) [reply site/zh-cn/tensorboard/scalars_and_keras.ipynb]: Thanks. i have already fixed.

Comment (lamberta) [reply site/zh-cn/tensorboard/scalars_and_keras.ipynb]: Not quite, I went ahead and fixed in https://github.com/tensorflow/docs-l10n/pull/212/commits/aa8ffb5f5e72af7537d1a94e8ef192b5e98e399e :)

Comment (ccw1996) [reply site/zh-cn/tensorboard/scalars_and_keras.ipynb]: Thanks a lot, i see these link careless.

**Pull request merged**